### DISCUSSION TITLE
vCard syncing

### DISCUSSION THREAD

**Discussion by avidseeker** [bitfireAT/icsx5#361] [Ideas]
ICSx5 is awesome for syncing and viewing my .ics files. It would be nice to have a similar app for .vcf (vCard) too, where it would synchronize a Android contacts with a .vcf file.

**Comment by ArnyminerZ** [comment]
Hi, while I agree this could be useful in some cases, I think it's strange to share contacts that way. In any case, you have DAVx5 which you can download at https://github.com/bitfireAT/davx5-ose and will synchronize contacts as well 😄

**Reply by avidseeker** [reply]
Here is a use case: use [khard](https://github.com/lucc/khard) to convert a maintained csv file of contacts into vcf that is then automatically imported to Android. The useful thing about ICSx5 is that it is *serverless*, there's no need to have a WebDav server to sync calendars. Having the same thing for contacts would be awesome.

**Reply by ArnyminerZ** [reply]
I don't think we are interested on working on this as of right now. Our main goal for ICSx⁵ is to make it as stable as possible so we can leave it as a low-maintenance app, for integrate its functionality directly in DAVx⁵.

We might reconsider it in the future, but I can't promise anything.

**Reply by 88-percent** [reply]
@avidseeker said:

> The useful thing about ICSx5 is that it is _serverless_, 

🡅 This is it for me. 

Another use case would be using Emacs org-vcard to create the vcf file and then pushing this to Android. I was doing this manually until recently.

I appreciate the devs have their focus, so it sounds like this isn't the right team for this one, but I agree this would be useful.
